src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] defaultArray;
+    private int size;
+
+    public ArrayList() {
+        defaultArray = (T[]) new Object[DEFAULT_CAPACITY];
+    }
+
+    private T[] getNewCapacityArray(T[] array) {
```

Comment:
private methods should be AFTER public ones.

And you do not need argument in this method. And do not need return anything - make it void.